Plugin/TrojanOrderPreventShippingAddress.php

```diff
@@ -83,4 +85,22 @@ public function beforeAssign(
 
         return [$cartId, $address];
     }
+
+    /**
+     *Compatible with Aheadworks One Step checkout and Product Labels
+     */
+    private function filterOutExtensionAttributes(array $data): array
+    {
+        foreach ($data as &$value) {
```

Comment:
@kapilpandya2203 Is there any reason you have used &$value ?

Reply:
I have used '&' so that the array which is passed is not coped and instead a reference of it is taken and further operations is done.Thus the function modifies the original array instead of a copy.